.github/workflows/CI.yml

```diff
@@ -0,0 +1,261 @@
+        echo "TARGET_CXX=aarch64-linux-gnu-cpp" >> "$GITHUB_ENV"
+        echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> "$GITHUB_ENV"
+    - name: Install armv7 cross compiler
+      if: matrix.platform.target == 'armv7-unknown-linux-gnueabihf'
+      run: |
+        sudo apt-get install -y gcc-arm-linux-gnueabihf libc6-armhf-cross libc6-dev-armhf-cross
+        echo "TARGET_CC=arm-linux-gnueabihf-gcc" >> "$GITHUB_ENV"
+        echo "TARGET_CXX=arm-linux-gnueabihf-cpp" >> "$GITHUB_ENV"
+        echo "CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER=arm-linux-gnueabihf-gcc" >> "$GITHUB_ENV"
+    - name: Install maturin
+      run: pip install maturin
+    - name: Build Wheels
+      run: |
+        maturin build -i python --release --out dist --no-sdist --target ${{ matrix.platform.target }} --manylinux ${{ matrix.platform.manylinux }}
+    - uses: uraimo/run-on-arch-action@v2.0.5
+      name: Install built wheel
+      with:
+        arch: ${{ matrix.platform.arch }}
+        distro: ubuntu18.04
+        githubToken: ${{ github.token }}
+        # Mount the dist directory as /artifacts in the container
+        dockerRunArgs: |
+          --volume "${PWD}/dist:/artifacts"
+        install: |
+          apt-get update
+          apt-get install -y --no-install-recommends python3 python3-pip
+          pip3 install -U pip
+        run: |
+          ls -lrth /artifacts
+          pip3 install cramjam --no-index --find-links /artifacts --force-reinstall
+          python3 -c "import cramjam"
+    - name: Upload wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheels
+        path: dist
+
+  pypy:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        default: true
+    - name: Build
+      run: cargo build
+    - name: Tests
+      run: cargo test
+    - uses: actions/setup-python@v2
+      with:
+        python-version: pypy-3.6
+    - name: Install maturin
+      run: pip install maturin
+    - name: Build Wheels
+      run: |
+        make pypy-build
+        ls -l dist
+    - name: Python UnitTest
+      run: |
+        pip install cramjam --no-index --find-links dist
+        pypy -c "import cramjam"
+    - name: Upload wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheels
+        path: dist
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags/')"
+    needs: [ macos, windows, linux, linux-cross, pypy ]
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: wheels
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Publish to PyPi
+        env:
+          TWINE_USERNAME: __token__
```

Comment:
`__token__`?

Reply:
https://pypi.org/help/#apitoken

Reply:
It was copied from other projects. I can revert to username/password if you prefer, but I think using apitoken is better.

Reply:
ah yes, just reading about it. Have added the PYPI_PASSWORD. Good now, thanks!